programmers/정렬.K번째수/sangkyu39.py

```diff
@@ -0,0 +1,7 @@
+def solution(array, commands):
+    answer = []
+    for cmd in commands:
+        newArr = array[cmd[0] - 1:cmd[1]]
+        newArr.sort()
+        answer.append(newArr[cmd[2] - 1 ])

```

Comment:
cmd를 for문에서 unpacking해서 i, j, k로 사용할 수도 있을 것 같아요!
cmd 값을 index로 접근하는 것보다 깔끔한 느낌인 것 같아서 개인적으로 선호하는 편입니다 ㅎㅎ

```suggestion
    for i, j, k in commands:
        newArr = array[i - 1:j]
        newArr.sort()
        answer.append(newArr[k - 1])
```